Readme.md

```diff
@@ -75,4 +79,8 @@ Shell profile configuration file, Shell command alias configuration file, PHP co
 ```
 
 ## Author 
-- [Valay Dave](valaygaurang@gmail.com)
\ No newline at end of file
+- [Valay Dave](valaygaurang@gmail.com)
+
```

Comment:
@inverse Please Add yourself as contributor too.

Reply:
Or we could remove this? It's clear from the repo name + new github UI

![image](https://user-images.githubusercontent.com/713525/90292024-61bac900-de81-11ea-9ec1-b629399e0bed.png)


Happy to just leave your name there too!

Reply:
Yes. Works.

Reply:
Lets keep your name :)